plancton/__init__.py

```diff
@@ -217,11 +217,17 @@ def _read_conf(self):
 
   # Set up monitoring target.
   def _influxdb_setup(self):
+    self.streamers = []
     if not self.conf["influxdb_url"]:
-      self.streamer = lambda *x,**y: True
+      self.streamers.append(lambda *x,**y: True)
```

Comment:
Hmmm this lambda is no longer necessary. Just remove this line, and `self.streamers` will be an empty list in case no `influxdb_url` is specified, no need to add a no-op streamer.